index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="./styles.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+    <nav>
+        <h1>IdeaBox</h1>
+    </nav>
+    <main>
+        <section>
+            <label for="title">Title:</label>
+            <input type="text" id="title" name="title">
+            <label for="body">Body:</label>
+            <input type="text" id="body" name="body">
+            <button>Save</button>
```

Comment:
Great use of semantic HTML tags! I wanted to suggest we wrap the labels, inputs and button in a form tag to take advantage of the built in methods available to form elements. That'll allow us to use the submit event listener and make it more accessible for screen readers as well.

Reply:
Great suggestion, Josh! Will update the HTML to include form tags.